hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -313,6 +313,13 @@ public RaftPeerId getPeerId() {
       List<RaftPeer> raftPeers = new ArrayList<>();
       // Add this Ratis server to the Ratis ring
       raftPeers.add(localRaftPeer);
+      for (SCMNodeDetails nodeDetails : details.getPeerNodeDetails()) {
```

Comment:
I think , we need to add the peers depending upon whether SCM needs to start as single node server(1st regular start) or participate in 3 node ring(subsequent restarts). Let's hold on to this change for now and i will take care of this in the bootstrap sequence.

Reply:
Sure sounds good.